core/iwasm/libraries/wasi-nn/src/wasi_nn_openvino.c

```diff
@@ -460,14 +460,14 @@ get_output(void *ctx, graph_execution_context exec_ctx, uint32_t index,
 
     CHECK_OV_STATUS(ov_tensor_get_byte_size(ov_tensor, &byte_size), ret);
 
-    if (byte_size > *output_tensor_size) {
+    if (byte_size > output_tensor->size) {
```

Comment:
May I ask about the difference between output_tensor->size and output_tensor_size?

Reply:
this PR makes output_tensor_size output-only.